test/e2e/main_test.go

```diff
@@ -27,6 +32,10 @@ var (
 type globalContext struct {
 	// numberOfNodes to be used for the test suite.
 	numberOfNodes int
+	// skipNodeDeletion flag allows skipping deletion of nodes within the test suite
```

Comment:
should this doc be consistent with above doc for skipNodeDeletion?

Reply:
no preference but I can change it.

Reply:
This would be just a nit.